### PR TITLE
Add documentation for the TLS paths

### PR DESCRIPTION
**Pull request by yago-123**

### CODE REVIEW

**Comment by yago-123** [comment]
@cpipilas anything that may be worth adding?

**Comment by yago-123** [comment]
> > @cpipilas anything that may be worth adding?
> 
> I'm missing some usage examples, specially for certs it would be really helpful.

I'm not sure what else would you expect aside from the example that I set. Is relatively straightforward this operation... Could you give any idea?

**Review by cpipilas — APPROVED** [review]

File: docs/integration/kubernetes.md [hunk 1]
```diff
@@ -823,3 +823,22 @@ Also, please make sure that your agent is [up to date](https://sematext.com/docs
 
 ### Why we need hostNetwork access and how to turn it off if desired
 Check out [our page about hostnetwork](https://sematext.com/docs/agents/sematext-agent/kubernetes/hostnetwork/).
+
+### Setting TLS Certificate Paths
```
Comment (cpipilas) [inline docs/integration/kubernetes.md:827]: @yago-123 for some reason I missed that section. We are good with this.

**Comment by cpipilas** [comment]
> > > @cpipilas anything that may be worth adding?
> > 
> > 
> > I'm missing some usage examples, specially for certs it would be really helpful.
> 
> I'm not sure what else would you expect aside from the example that I set. Is relatively straightforward this operation... Could you give any idea?



> > > @cpipilas anything that may be worth adding?
> > 
> > 
> > I'm missing some usage examples, specially for certs it would be really helpful.
> 
> I'm not sure what else would you expect aside from the example that I set. Is relatively straightforward this operation... Could you give any idea?

We are good: https://github.com/sematext/docs/pull/522#discussion_r1312915375

File: docs/integration/kubernetes.md (hunk 1 above)
Comment (yago-123) [reply docs/integration/kubernetes.md:827]: OK, asking for review for the final boss then

File: docs/integration/kubernetes.md [hunk 2]
```diff
@@ -823,3 +823,22 @@ Also, please make sure that your agent is [up to date](https://sematext.com/docs
...
+While monitoring Kubernetes master components, the Sematext agent will automatically retrieve certificates from the host machine.
```
Comment (otisg) [inline docs/integration/kubernetes.md:828]: Sematext agent ==> Sematext Agent (we treat it as a name elsewhere, I think?)

**Review by otisg — APPROVED** [review]
LGTM

File: docs/integration/kubernetes.md (hunk 2 above)
Comment (yago-123) [reply docs/integration/kubernetes.md]: Modified, also checked if there were more cases like this, didn't found places in which we don't use Sematext Agent

**Pull request merged**